lib/utils/firebase_services.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:community/utils/firebase_storage_services.dart';
+import 'dart:io';
+
+
+
+class FirebaseService {
+  final FirebaseFirestore _firestore = FirebaseFirestore.instance;
+  final FirebaseStorageService _storageService = FirebaseStorageService();
+
+  Future<bool> uploadUserData(Map<String, dynamic> userData) async {
+    try {
+      final user = FirebaseAuth.instance.currentUser;
+      if (user != null) {
+        final userFirebaseUid = user.uid;
+        await _firestore.collection('users').doc(userFirebaseUid).set(userData);
+        return true;
+      }
+      return false; // No user is signed in.
+    } catch (e) {
+      print('Error uploading data to Firestore: $e');
+      return false;
+    }
+
+  }
+  Future<bool> updateUserBio(String userFirebaseUid, String userBio) async {
+    try {
+      final userRef = _firestore.collection('users').doc(userFirebaseUid);
```

Comment:
Keep all table and database in new Constant Files

appdatabase/model/DbConstants

Reply:
Should I keep them in one file ie firebase_services.dart or creat a seperate file for each required method?

Reply:
it should ne new folder,
  appdatabase/tables/UserTableConstant